plans/store-list.md

```
@@ -0,0 +1,313 @@
+- Super 1: Pacific NW
+- Stop & Shop (MA, RI, CT, NY, NJ)
+- The Fresh Market
+  Weis (PA, NY, NJ, MD, VA, DE)
+- Whole Foods Market Mid-Atlantic Division (OH, PA, MD, VA, DC, Southern NJ
+- Yoke’s: Pacific NW
+- Most natural grocers in Hawaii
+- Sprouts Farmers Market (Nationwide)
+
+## **La Famiglia DelGrosso Pasta and Pizza Sauces**
+
+#### Alabama
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)